get_weather_Utils.py

```diff
@@ -0,0 +1,84 @@
+from decimal import *
+import boto3
+import requests
+from datetime import date, timedelta, datetime
+import formatting_Utils
+
+
+
+def make_url_api(location_id, day_str):
+    url = "https://www.metaweather.com/api/location/" + location_id + "/"+ day_str  + "/"
+    print("API-URL aus Funktion make_url_api: ", url)
+    return url
+
+
+def get_location(number):
+    dynamodb = boto3.resource('dynamodb')
+    table_destination = dynamodb.Table('destination')
+    response = table_destination.scan()
```

Comment:
scan is very expensive. 
Currently you load all data that you have. 
would be better to search for a specific location

Reply:
🤔 Will look into that. 
It's a small Table of "city" and "city_id". Want to get weather for all of them.